vault.py

```diff
@@ -174,73 +176,230 @@ def api_vault_republish(ctx: rule.Context, coll: str) -> api.Result:
 
 @api.make()
 def api_vault_copy_to_research(ctx: rule.Context, coll_origin: str, coll_target: str) -> api.Result:
-    """Copy data package from vault to research space.
+    """Copy data collection from vault to research space.
+    The data collection will be copied into a subfolder of coll_target.
 
     :param ctx:         Combined type of a callback and rei struct
-    :param coll_origin: Collection of data package to copy
-    :param coll_target: Collection to copy data package to
+    :param coll_origin: Origin data collection in vault space
+    :param coll_target: Target collection path in research space
 
     :returns: API status
     """
+    # Validate target location
     zone = user.zone(ctx)
+    if coll_target == f"/{zone}/home":
+        return api.Error('HomeCollectionNotAllowed',
+                         'Please select a specific research folder for your datapackage')
 
-    # API error introduces post-error in requesting application.
-    if coll_target == "/" + zone + "/home":
-        return api.Error('HomeCollectionNotAllowed', 'Please select a specific research folder for your datapackage')
-
-    # Check if target is a research folder. I.e. none-vault folder.
+    # Check target is not in vault space
     parts = coll_target.split('/')
-    group_name = parts[3]
-    if group_name.startswith('vault-'):
-        return api.Error('RequiredIsResearchArea', 'Please select a specific research folder for your datapackage')
+    if len(parts) > 3 and parts[3].startswith('vault-'):
```

Comment:
Can we rewrite this condition using [pathutil.info](https://github.com/UtrechtUniversity/yoda-ruleset/blob/development/util/pathutil.py#L74)?